### PR TITLE
doc: make it easier to reach the old dev doc TOC

### PR DESCRIPTION
**Pull request by smithfarm**
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
"Ceph internals" sounds good to me!

@smithfarm thanks! probably, intead of renaming it to `doc/dev/index-toc.rst`, we could use something like `doc/dev/internals.rst` ?

**Comment by smithfarm** [comment]
@tchaikov Good idea. Done.

**Comment by tchaikov** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
Changelog:

* s/Ceph internals/Ceph Internals/ in top-level TOC
* change title of internals.rst from "Internal Developer Documentation" to "Ceph Internals" (to match TOC)

**Comment by smithfarm** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
doc render LGTM

**Comment by dmick** [comment]
Bravo.  :100:

**Comment by dmick** [comment]
The only thing that concerns me a bit is that comment that's changing: "pre-2016".  Does that mean that some of this doc is out of date?

**Comment by smithfarm** [comment]
@dmick When I randomly do `git log $FILENAME` on files in `doc/dev/*`, I see some that have not been updated since before 2016. That's why I originally wrote "pre-2016".

File: doc/dev/internals.rst [hunk 1]
```diff
@@ -1,8 +1,8 @@
 :orphan:
```
Comment (tchaikov) [inline doc/dev/internals.rst]: nit, could drop this `:orphan:` directive.

**Comment by smithfarm** [comment]
jenkins render docs

**Pull request merged**